project/BuildHelper.scala

```diff
@@ -166,14 +150,12 @@ object BuildHelper {
 
   def crossPlatformSources(scalaVer: String, platform: String, conf: String, baseDir: File) = {
     val versions = CrossVersion.partialVersion(scalaVer) match {
-      case Some((2, 11)) =>
-        List("2.11+", "2.11-2.12")
       case Some((2, 12)) =>
-        List("2.11+", "2.12+", "2.11-2.12", "2.12-2.13")
+        List("2.12", "2.12+", "2.12-2.13")
```

Comment:
`2.12` is still used for https://github.com/zio/zio-prelude/pull/1175/files#diff-be30bee277855f18960ddd438b1594a798cbaf405b6e0ab9d4d7eccf3d965638

This class was in `2.11-2.12`. I transferred it to this `2.12` directory